sqlite_test.go

```diff
@@ -1539,3 +1539,32 @@ func TestConnLogger_read_tx(t *testing.T) {
 		}
 	}
 }
+
+func TestExpandedSQL(t *testing.T) {
```

Comment:
I guess it's too tedious to make this look for leaks, eh?

(probably not visible to https://pkg.go.dev/runtime#ReadMemStats ...)